### DISCUSSION TITLE
Multiples Idea that I have thinking from my head

### DISCUSSION THREAD

**Discussion by Sewlell** [ppy/osu#22431] [Ideas] [answered]
Anyway here is several idea that I think it will make the game more or less modern / multi-modes or something.

**1. Redesign of Leaderboard and Map Details UI with more angled slope.**
I already see developers tries to do the design stuff on the lazer updates video uploaded by osu! but yea

**2. Enabling leaderboard and multiplayer for fanmade rulesets.**
Uh... maybe? Not a programmer and I am not sure whether it is working. And personally I think it need a some sort of permission from the ruleset(s) creator and maybe a better server (If the osu!supporter was pushed enough for this kind of crowdfunding). I also think of a kind of mechanic that detects whether if you have the specific ruleset when you trying to join the specific ruleset match in multiplayer.

**3. Enabling skins creators to customize even more stuff in the game.**
It will be the most trickiest stuff to do. Like in the osu!stable, skin creators will be able to create a full fancy png on the menu UI or even a video behind. I expected you can like customize something like the Difficulty Background ( that showing bunch of triangle by default ), Low Bar ( the bar that show up mods, random, and options ) and etc.

Of course I think this is going to be like crazy for computer storage (lets say 500MB without compression on videos or image on a single skin and if all customization bits were customized?), but I think that is not a big deal while 40% of peoples may have 1TB or even 2TB storage now.

Well here maybe all my abridged ideas on osu!lazer. Pretty sure this is going to be hellish for devs.

**Comment by bdach** [comment, marked as answer]
1. https://github.com/ppy/osu/issues/22204
2. https://github.com/ppy/osu/discussions/15157
3. https://github.com/ppy/osu/discussions?discussions_q=label%3Aarea%3Askinning
